src/components/Image.js

```diff
@@ -9,6 +9,13 @@ import { imageHeight, imageWidth } from '../helpers';
 import { SettingsContext } from '../SettingsProvider';
 import { ImageMessage } from './ImageMessage';
 import { ImageRights } from './ImageRights';
+import { useInterval } from '../hooks';
+
+const addQueryParam = (url, param) => {
+  if (!url?.length) return;
+
+  return url.includes('?') ? `${url}&${param}` : `${url}?${param}`;
```

Comment:
there could be the case, where the `url` ends with `/`, right? maybe we need to catch that, because `.../?...` can behave differently to `...?...`. i do not know how to test right now. let's go with this solution until we face a problem.

Reply:
if the url ends on a slash, that slash is removed now, as it was not really difficult to do so.